src/util.h

```diff
@@ -90,7 +90,7 @@ constexpr int compare_nocase(std::string_view str1, std::string_view str2)
 // Efficiently assign format to std like string.
 // The output string will always fit the formatted string in its entirety.
 template<typename String, typename... Args>
-void assign_format(String &output, fmt::format_string<Args...> format, Args &&... args)
+void assign_format(String &output, fmt::format_string<Args...> format, Args &&...args)
```

Comment:
The formatting has changed here unnecessarily.

Reply:
My formatter runs on save, so these things are annoying but only once - unless you use a different linter/formatter.

Reply:
OmniBlade has told us to use the clang-format 10.0.0.0 which is part of LLVM. You can install that and manually link clang-format 10.0.0.0 in your IDE if it allows to do that (Visual Studio for Windows does).